ci/snap/bootstrap/snapcraft.yaml

```diff
@@ -8,8 +8,11 @@ grade: stable
 confinement: classic
 base: core22
 issues: https://bugs.launchpad.net/ubuntu-desktop-provision/+filebug
-contact: https://bugs.launchpad.net/ubuntu-desktop-provision/+filebug
-
+contact: https://github.com/canonical/ubuntu-desktop-provision/issues
```

Comment:
Didn't we want to use the launchpad bug tracker as an outside facing point of contact?

Reply:
I agree for issues but I wondered if for contact we sent folks to github or even provide an email address? Ie if someone is on snap store and clicks contact us sending them to launchpad might not quiet be what we want. 

WDTY?

Reply:
What does "contact" mean in this case anyways? I'd just like to avoid having bug reports spread across github and launchpad again, since it's chaotic enough already with old issues from ubuntu-desktop-installer.

Reply:
Did some investigating. The BLUF is that I think we should drop both contact and issue from the snapcraft.yaml 
 because they're misleading. The fields themselves are described [here](https://snapcraft.io/docs/snapcraft-yaml-reference). Okay the nitty gritty:

Afaict `contact` is displayed in three places: `snap info <name>`, the App Center, and snapstore.io. The values set in the snapstore.io publisher page overrides the snapcraft.yaml. 

On `issues` I couldn't find the `snapcraft.yaml` value anywhere. If you set it in the snapstore.io publisher page then the only place I could find it was on the snapstore.io entry under "Report a bug".